lib/xenops_server.ml

```diff
@@ -1400,13 +1401,12 @@ and perform ?subtask ?result (op: operation) (t: Xenops_task.t) : unit =
 			debug "VM.receive_memory %s power_state = %s" id (state.Vm.power_state |> rpc_of_power_state |> Jsonrpc.to_string);*)
 
 			(* set up the destination domain *)
-			perform_atomics (
-				simplify [VM_create (id, Some memory_limit);] @
-				(* Perform as many operations as possible on the destination domain before pausing the original domain *)
-				(atomics_of_operation (VM_restore_vifs id))
-			) t;
-
 			(try
+				perform_atomics (
```

Comment:
Hold on... I may have been wrong about that, sorry. Perhaps sending `Handshake.Error` is the right thing to do!